### DISCUSSION TITLE
create MV using Internal Source cross database in one RisingWave cluster.

### DISCUSSION THREAD

**Discussion by Murasame233** [risingwavelabs/risingwave#19705] [Q&A]
I have two dbs in one RisingWave cluster. And I want to create a MV with some table of these two dbs.

I am using the PostgreSQL connector for this situation. And is there any solution like internal connector for this?

**Comment by cloudcarver** [comment]
Hi @Murasame233 , the cross-database materialized view is under active development. More details can be found in [this issue](https://github.com/risingwavelabs/risingwave/issues/19631)
